src/test/datascience/editor-integration/codewatcher.unit.test.ts

```diff
@@ -560,6 +560,81 @@ testing2`;
         document.verifyAll();
     });
 
+    test('Test two command ordering correct ordering', async () => {
```

Comment:
Usually I like our vscode tests over unit, but one of these tests actually caught an issue in an earlier checkin. And the way they are set up they actually work pretty good for testing this scenario. I put this test in the old bits and it failed with the expected failure (trying to interleave instead of running each in order).

![image](https://user-images.githubusercontent.com/812783/142296171-f746bfae-11b5-437e-b100-6e3a5eaa6545.png)